src/antares/service/api_services/run_api.py

```
@@ -0,0 +1,89 @@
+from antares.service.base_services import BaseRunService
+
+
+class RunApiService(BaseRunService):
+    def __init__(self, config: APIconf, study_id: str):
+        super().__init__()
+        self.config = config
+        self.study_id = study_id
+        self._base_url = f"{self.config.get_host()}/api/v1"
+        self._wrapper = RequestWrapper(self.config.set_up_api_conf())
+
+    def run_antares_simulation(self, parameters: Optional[AntaresSimulationParameters] = None) -> Job:
+        url = f"{self._base_url}/launcher/run/{self.study_id}"
+        try:
+            if parameters is not None:
+                payload = parameters.model_dump()
+                response = self._wrapper.post(url, json=payload)
+            else:
+                response = self._wrapper.post(url)
+            job_id = response.json()["id"]
+            return self._get_job_from_id(job_id)
+        except APIError as e:
+            raise AntaresSimulationRunningError(self.study_id, e.message) from e
+
+    def _get_job_from_id(self, job_id: str) -> Job:
+        url = f"{self._base_url}/launcher/jobs/{job_id}"
+        response = self._wrapper.get(url)
+        job_info = response.json()
+        launcher_params = json.loads(job_info["launcher_params"])
+        return Job(job_id, job_info["status"], launcher_params["auto_unzip"])
+
+    def wait_job_completion(self, job: Job, time_out: int) -> None:
+        start_time = time.time()
+        repeat_interval = 5
+        while job.status in (JobStatus.RUNNING.value, JobStatus.PENDING.value):
+            if time.time() - start_time > time_out:
+                raise SimulationTimeOutError(job.job_id, time_out)
+            time.sleep(repeat_interval)
+            updated_job = self._get_job_from_id(job.job_id)
+            job.status = updated_job.status
+            job.unzip_output = updated_job.unzip_output
+
+        if job.unzip_output:
```

Once you go out of the loop your status could be success or failed.
If it's failed, you should raise an Exception to say the simulation failed.
Else you can continue your code